Automattic/wp-calypso
# Full Site Editing: confirm changes to template parts

#### Changes proposed in this Pull Request

* Add a post-update confirmation when updating a template part

![in-action](https://user-images.githubusercontent.com/195089/59468558-ff3ae880-8df7-11e9-82f7-f658dbee8548.gif)

#### Testing instructions

* load an existing Template Part post
* update it
* ensure that a confirmation panel appears after clicking the button
* since the button is hacked into place atop the "real" update button, try it at various screen sizes, devices, etc

Fixes #33629
